docs/content/faq/how-can-i-cancel-a-crashed-deploy.md

```diff
@@ -0,0 +1,3 @@
+# How can I cancel a crashed deploy?
+
+Flynn will automatically detect a crashed deploy. You can pick a new revision to deploy, or change your application environment and re-attempt the same revision.
```

Comment:
I am not really sure what is meant by a "crashed deploy", are we talking about `git push flynn master` failing, or are we talking about successfully pushing code but the resulting jobs crash?